examples/ansible/inventory/ansibleRun-executable-inventory.yml

```diff
@@ -0,0 +1,28 @@
+apiVersion: ansible.crossplane.io/v1alpha1
+kind: AnsibleRun
+metadata:
+  name: executable-inventory
+spec:
+  forProvider:
+    executableInventory: True
+    inventoryInline: |
+      #!/usr/bin/env python
```

Comment:
it seems not working with `python2` , so may be adding the python 3 shebang `#!/usr/bin/python3`